docker/development/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.7'
+
+networks:
+    books:
+        name: books-external
+
+services:
+    client-builder:
+        build:
+            context: ../..
+            dockerfile: ./docker/development/Dockerfile
+        networks:
+            - books
+        volumes:
+            - ../..:/app
+        working_dir: /app
+        ports:
+            - 8080:8080
+        command: './tools/run-developmentcd.sh'
```

Comment:
I don't like this tbh. What you are doing here is build image with python image in the Dockerfile and then just installing needed packages and running client in node environment. IMO much better approach would be something like this:
```yaml
services:
    client-builder:
        image: node:12
        working_dir: /app
        volumes:
            - "../..:/app"
        networks:
            - books
        ports:
            - 8080:8080
        command:
            - "/app/tools/run-development.sh"
```

Then you don't have to create Dockerfile and install node by yourself within python image.